electron/main.js

```diff
@@ -108,6 +100,21 @@ if (process.platform === 'win32') {
   ipcMain.on('wrapper-restart', function() {
     squirrel.installUpdate();
   });
+
+  if (shouldQuit) {
+    // Stop further execution on update to prevent second tray icon
+    return;
+  }
+}
+
+///////////////////////////////////////////////////////////////////////////////
+// Fix indicator icon on Unity
+// Source: https://bugs.launchpad.net/ubuntu/+bug/1559249
+///////////////////////////////////////////////////////////////////////////////
+if (process.platform === 'linux') {
```

Comment:
Instead of having all these `if`'s here and having a comment with `Fix indicator icon on Unity` we should consider having a function called `fixIndicatorIconOnUnity` or something like that. :)